### DISCUSSION TITLE
Have a WeChat Group in gravitino？

### DISCUSSION THREAD

**Discussion by luhea** [apache/gravitino#5684] [Q&A] [answered]
Have a WeChat Group in gravitino？it advantageous to  promotion gravitino

**Comment by yuqi1129** [comment, marked as answer]
Yes, there is the QR code of the WeChat Group, This is one of Gravitino's WeChat groups that has a few members due to the limitations of Wechat.  

![image](https://github.com/user-attachments/assets/8472263b-e49c-4a54-a0dc-6c2192f5023a)

**Comment by melin** [comment]
@yuqi1129 过期了

**Comment by xunliu** [comment]
Please add my WeChat(`liuxun-ASF`), I will be inviting you to join the WeChat group.
